source/acquaman/AMEXAFSScanActionControllerAssembler.cpp

```diff
@@ -213,10 +213,12 @@ AMAction3 *AMEXAFSScanActionControllerAssembler::generateActionTreeForEXAFSStepA
 
 			for(int x = 0; x < detectors_->count(); x++){
 
-				AMAction3 *detectorSetDwellAction = detectors_->at(x)->createSetAcquisitionTimeAction(variableIntegrationTimes.at(i));
+				if(detectors_->at(x)->supportsSynchronizedDwell()) {
```

Comment:
Yeah, it seems as if the check will have to be whether the read method is request read.